jasypt/src/main/java/org/jasypt/encryption/pbe/StandardPBEStringEncryptor.java

```diff
@@ -743,12 +743,15 @@ public String decrypt(final String encryptedMessage) {
             return new String(message, MESSAGE_CHARSET);
         
         } catch (EncryptionInitializationException e) {
+        	System.out.println("Error while decrypting the following message: \"" + encryptedMessage + "\"");
```

Comment:
@prakashiit  try to reuse utils like CLIUtils.showError

Reply:
@SriMouli I've changed the `System.out.println` to `System.err.println` after your suggestion but I don't want to alter any existing message that these exceptions may contain. Also the logs that I'm printing here are not actually errors but are only there to serve as a marker. Since the file being encrypted/decrypted could potentially contain a large no. of lines, I thought it would be helpful if we could just tell our users exactly at which line in their file did the code fail. Besides, jasypt is designed to hide error messages of `EncryptionOperationNotPossibleException` and two of the three logs I've added in this file will fall under that category itself.

After your recommendation though, I've refrained from myself printing the errors everywhere else in the code and have used CLIUtils.showError which is being called in the main method.